dongkyu/initial-settings-express/myapp/app.js

```diff
@@ -0,0 +1,139 @@
+const http = require("http");
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const mysql = require("mysql2");
+const dotenv = require("dotenv");
+dotenv.config();
+
+const { DataSource } = require("typeorm");
+const { allowedNodeEnvironmentFlags } = require("process");
+
+const myDataSource = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+myDataSource.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
```

Comment:
typeorm 공식 문서를 확인해보시면 dataSource를 initialize()를 사용할 때엔 `.then() ~ .catch()` 부분을 개행처리하여 가독성을 높여주고 있습니다.

아래 코드를 참고하여 코드를 수정해주세요!
```suggestion
myDataSource.initialize()
  .then(() => {
    console.log("Data Source has been initialized!");
  })
  .catch(() => {
    console.log("Failed to connect Database");
  })
```